java/core/src/main/java/sleeper/core/properties/instance/DefaultProperty.java

```diff
@@ -347,11 +347,11 @@ public interface DefaultProperty {
             .propertyGroup(InstancePropertyGroup.DEFAULT).build();
     UserDefinedInstanceProperty DEFAULT_LAMBDA_CONCURRENCY_MAXIMUM = Index.propertyBuilder("sleeper.default.lambda.concurrency.max")
             .description("Default value for the maximum concurrency for each lambda within the Sleeper instance. " +
-                    "By default the maximum concurrency is unlimited. Each lambda also has its own property that " +
+                    "By default the maximum concurrency is set 10 concurrent tables. Each lambda also has its own property that " +
```

Comment:
I think this would read better as "is set to 10 which is enough for 10 online tables. If there are more online tables this number may need to be increased".

Reply:
Addressed